packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -718,7 +718,7 @@ curl -v -X POST \
 }
 ```
 
->**Note:** The `type` property and the `schema` and `type` links will be present in all responses if the User Types feature is enabled, whether or not the user is created with a non-default User Type. See [User object](#user-object).
+>**Note:** The `type` property and the `schema` and `type` links will be present in all responses, whether or not the user is created with a non-default User Type. See [User object](#user-object).
```

Comment:
Do we need this note anymore? I believe this was more to specify the behavior of flag on vs flag off, and that distinction no longer exists.

Reply:
👍

Reply:
Yeah, I wondered about that, but ultimately decided the bit about 'schema' and 'type' links was marginally worth keeping.  I'm fine removing this.